zingolib/src/lightclient/send.rs

```diff
@@ -405,20 +414,166 @@ pub mod send_with_proposal {
             .await;
         }
 
-        #[ignore = "live testnet"]
+        #[ignore = "live testnet: testnet relies on NU6"]
         #[tokio::test]
         /// this is a live sync test. its execution time scales linearly since last updated
+        /// note: live send waits 2 minutes for confirmation. expect 3min runtime
         async fn testnet_cbbhrwiilgbrababsshsmtpr_shield_hot() {
-            std::env::set_var("RUST_BACKTRACE", "1");
-            let client = sync_example_wallet(ExampleWalletNetwork::Testnet(
-                ExampleTestnetWalletSeed::CBBHRWIILGBRABABSSHSMTPR(
+            // install default crypto provider (ring)
+            if let Err(e) = rustls::crypto::ring::default_provider().install_default() {
+                log::error!("Error installing crypto provider: {:?}", e)
+            };
+
+            let case =
+                ExampleWalletNetwork::Testnet(ExampleTestnetWalletSeed::CBBHRWIILGBRABABSSHSMTPR(
                     ExampleCBBHRWIILGBRABABSSHSMTPRVersion::G2f3830058,
-                ),
-            ))
-            .await;
+                ));
+
+            let client = sync_example_wallet(case).await;
 
             with_assertions::propose_shield_bump_sync(&mut LiveChain::setup().await, &client, true)
                 .await;
         }
+
```

Comment:
So should we ignore this?   It shouldn't be part of the automated test suite, should it?

Reply:
yeahh